databases_preview.go

```diff
@@ -0,0 +1,125 @@
+package scalingo
+
+import (
+	"context"
+	stderrors "errors"
+
+	"github.com/Scalingo/go-utils/errors/v2"
+)
+
+const databasesPreviewResource = "databases"
```

Comment:
```suggestion
const databasesResource = "databases"
```
In the constant there is nothing about the preview, so I would just simplify the name.